skimage/measure/_regionprops.py

```diff
@@ -835,6 +846,133 @@ def regionprops(label_image, intensity_image=None, cache=True):
     return regions
 
 
+def euler_number(image, neighbourhood=8):
+    """Calculate the Euler characteristic in 2D binary image, that characterize
+    the topology of the objects.
+    
+    A neighbourhood configuration is constructed (see variable config), 
```

Comment:
I would remove "see variable config" since we don't expect users reading the docstring to read the code as well. However you can move this to a comment within the function.

Reply:
ok